### PR TITLE
Update conv.jl

### PR DESCRIPTION
**Pull request by rkube**
Clarify meaning of `pad` parameter in `Conv`.

### CODE REVIEW

**Review by ToucheSir — DISMISSED** [review]
Thanks!

File: src/layers/conv.jl [hunk 1]
```diff
@@ -50,7 +50,7 @@ Then:
 * `filter` should be a tuple of `N` integers.
 * Keywords `stride` and `dilation` should each be either single integer,
   or a tuple with `N` integers.
-* Keyword `pad` can be:
+* Keyword `pad` specifies the number of elements added to the image borders. It can be
```
Comment (DhairyaLGandhi) [inline src/layers/conv.jl:53]:
```suggestion
* Keyword `pad` specifies the number of elements added to the borders of the data array. It can be
```

**Review by DhairyaLGandhi — DISMISSED** [review]
Lgtm

**Review by DhairyaLGandhi — APPROVED** [review]
bors r+

thanks!

**Pull request merged**